### ISSUE TITLE
Emulate Ctrl-F KeyDown event with DevTools method

### ISSUE THREAD

**Issue by AStrelkov** [MicrosoftEdge/WebView2Feedback#1278]
I try to show FindString dialog by sending (emulating) Ctrl-F keys pressed to WebView2. When WebView2 has focus I call **CallDevToolsProtocolMethod**(L"Input.dispatchKeyEvent", L"{\"type\":\"keyDown\", \"modifiers\":2,\"unmodifiedText\":\"F\"}", ...), but nothing happened on screen and no errors raising. What I am doing wrong?

VC++ 2017
Windows 7 64bit

**Comment by novac42** [comment]
Hi @AStrelkov do you still have this issue with the latest runtime and SDK?

**Comment by AStrelkov** [comment]
Yes, nothing happens. WebView2 ver. 1.0.1823.32. I have tried many options, but without success. Last option was: 
L"{"type":"keyDown","modifiers":2,"windowsVirtualKeyCode":70,"code":"KeyF","unmodifiedText":"F"}"

**Comment by novac42** [comment]
@AStrelkov Is it only happening in Win7? Do you have same issue on Win10+ OS?

**Comment by AStrelkov** [comment]
Yes. I have same issue on Windows 11 Pro OS and WebView2 ver. 1.0.1901.177.

**Comment by novac42** [comment]
@AStrelkov Thanks for the clarification! Which framework you are using (Win32/WPF/WinUI...)?

**Comment by AStrelkov** [comment]
Win32, VC++ 2017 Express Edition
